testing/benchmark/Makefile

```diff
@@ -6,7 +6,7 @@ TFVARS_SOURCE ?= terraform.tfvars.example
 
 BENCHMARK_WARMUP_TIME ?= 5m
 BENCHMARK_AGENTS ?= 64
-BENCHMARK_COUNT ?= 3
+BENCHMARK_COUNT ?= 6
```

Comment:
We should also double the benchmark timeout to 8h here: https://github.com/elastic/apm-server/blob/30f2301fd67082fefd0f59aedf5055df72b7f691/.ci/benchmarks.groovy#L22

Reply:
8h :flushed:

OK

Reply:
yeah... That would be the worst case scenario. In practice it's unlikely that the benchmarks run for more than 3 hours, unless the first run fails and the retry runs for the full length.